frontend/src/pages/projects/screens/detail/pipelines/PipelinesList.tsx

```diff
@@ -9,11 +9,19 @@ import { usePipelinesAPI } from '~/concepts/pipelines/context';
 import EmptyStateErrorMessage from '~/components/EmptyStateErrorMessage';
 import { TABLE_CONTENT_LIMIT, LIMIT_MAX_ITEM_COUNT } from '~/concepts/pipelines/const';
 
-const PipelinesList: React.FC = () => {
+interface PipelinesListProps {
+  setIsPipelinesEmpty: (arg0: boolean) => void;
+}
```

Comment:
While there are a few instances of `interface` in the repo, we tend to lean towards `type` definitions. Also replace `arg0` with an appropriate name.

```suggestion
type PipelinesListProps = {
  setIsPipelinesEmpty: (isEmpty: boolean) => void;
}
```